### PR TITLE
chore(dependabot) Create dependabot.yml

### PR DESCRIPTION
**Pull request by ArjunSharda**
# Summary
This will make a dependabot.yml file which will scan for latest versions of the github actions used in this repository.

### CODE REVIEW

**Comment by ArjunSharda** [comment]
Please review @jackkav

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```
Comment (filfreire) [inline .github/dependabot.yml]: I think we could run this daily, similar to [GH's official suggestion](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot):
```suggestion
      interval: "daily"
```

**Review by filfreire — APPROVED** [review]
Hi @ArjunSharda Thank you for contributing this! I've added 1 suggestion, let me know what you think.

@DMarby - seeing as we have been updating these "reactively", I reckon it doesn't hurt if we add this GH workflow. More info about it [here](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot). What do you think?

**Comment by ArjunSharda** [comment]
Bumping for tomorrow

**Comment by jackkav** [comment]
Closing for now, we are considering a new approach to dependency management. Thanks for you contribution.

**Pull request closed without merging**